docs/source/install/ewc_ha.rst

```diff
@@ -0,0 +1,286 @@
+
+  # Replace `<EWC_LICENSE_KEY>` with a real license key, obtained in Email
+  helm install --set secrets.st2.license=<EWC_LICENSE_KEY> stackstorm/stackstorm-enterprise-ha
+
+.. note::
+    Don't have StackStorm Enterprise License?
+
+    Request a 90-day free trial at https://stackstorm.com/#product
+
+Once the deployment is finished, it'll show you first steps how to start working with the new cluster via WebUI or st2 client:
+
+.. figure :: /_static/images/helm-chart-notes.png
+    :align: center
+
+
+The installation uses some unsafe defaults which we recommend you change thoughtfully for production use via Helm ``values.yaml``.
+
+Helm values.yaml
+________________
+Helm package ``stackstorm-enterprise-ha`` comes with the default settings in ``values.yaml``.
+Fine-tune them to achieve desired configuration for the StackStorm Enterprise HA K8s cluster.
+
+You can configure:
+
+- number of replicas for each component
+- st2 auth secrets
+- st2.conf settings
+- RBAC roles, assignments and mappings
+- custom st2 packs and its configs
+- st2web SSL certificate
+- SSH private key
+- K8s resources and settings to control pod/deployment placement
+- configuration for Mongo, RabbitMQ clusters
+- configuration for in-cluster Docker registry
+
+.. warning::
+    It's highly recommended to set your own secrets as file contains unsafe defaults like self-signed SSL certificates, SSH keys, StackStorm access credentials and MongoDB/RabbitMQ passwords!
+
+Upgrading
+_________
+Once you make any changes to Helm values, upgrade the cluster:
+
+.. code-block:: bash
+
+  helm repo update
+  helm upgrade <release-name> stackstorm/stackstorm-enterprise-ha
+
+It will redeploy components which were affected by the change, taking care about keeping
+desired number of replicas to sustain every service alive during the rolling upgrade.
+
+
+Tips & Tricks
+_____________
+Save custom Helm values you want to override in a separated file, upgrade the cluster:
+
+.. code-block:: bash
+
+  helm upgrade -f custom_values.yaml <release-name> stackstorm/stackstorm-enterprise-ha
+
+Get all logs for entire StackStorm cluster with dependent services for Helm release:
+
+.. code-block:: bash
+
+  kubectl logs -l release=<release-name>
+
+Grab all logs only for stackstorm backend services, excluding st2web and DB/MQ/etcd:
+
+.. code-block:: bash
+
+  kubectl logs -l release=<release-name>,tier=backend
+
+
+Custom st2 packs
+----------------
+To follow the stateless model, shipping custom st2 packs is now part of the deployment process.
+It means that ``st2 pack install`` won't work in a distributed environment and you have to bundle all the
+required packs into a Docker image that you could codify, version, package and distribute in a repeatable way.
+The responsibility of such Docker image is to hold pack content and their virtualenvs.
+So custom st2 pack Docker image you have to build is just read-only dirs that are shared with the corresponding
```

Comment:
The custom st2 pack docker image you have to build is essentially a couple read-only directories that are shared with the corresponding st2 services in the cluster.